src/Conversion/AutoConversion/AutoConvertStream.tsx

```diff
@@ -7,22 +7,23 @@ import './AutoConvertStream.css';
 type Props = {};
 type State = {
   text: string,
+  output: string,
 };
 
 type SavedState = {
   text: string,
 };
 
 class AutoConvertStream extends LocalStorageComponent<Props, State, SavedState> {
-  private _str: string;
-  private _converted: string;
+  private _text: string = '';
```

Comment:
On the first visit to the page, before any state had been saved, this ended up undefined. Initializing it here allows the page to load successfully.